frontend/views/Home.vue

```diff
@@ -55,7 +55,7 @@
 
 <script>
 import sbp from '../../shared/sbp.js'
-import { LOAD_MODAL, CLOSE_MODAL } from '../utils/events'
+import { LOAD_MODAL, UNLOAD_MODAL } from '../utils/events'
```

Comment:
This should use the new `@utils/` shortcut right? Also, every import should specify its extension, so in this case `.js` should be added to the end (I know it didn't have that before, but it's something that should be fixed project-wide as we go)